src/features/broker-protection/actions/build-url-transforms.js

```diff
@@ -133,13 +129,8 @@ function processPathname (pathname, action, userData) {
  * @param {Record<string, string|number>} userData
  */
 export function processTemplateStringWithUserData (input, action, userData) {
-    /**
-     * Note: this regex covers both pathname + query params.
-     * This is why we're handling both encoded and un-encoded.
-     */
-    return String(input).replace(/\$%7B(.+?)%7D|\$\{(.+?)}/g, (match, encodedValue, plainValue) => {
-        const comparison = encodedValue ?? plainValue
-        const [dataKey, ...transforms] = comparison.split('|')
+    return String(input).replace(/\$\{(.+?)}/g, (_, value) => {
+        const [dataKey, ...transforms] = value.split('|')
```

Comment:
I'm missing the context of this bug. This code was able to handle encoded curly braces, so it's not clear why we need a full decoding above.
Is the bug that we started getting encoded characters other than `{}`?